configfiles/datadrivenfprompt/config_Dplus_PromptFrac_pp5TeV.yml

```diff
@@ -11,7 +11,8 @@ efficiencies:
         feeddown: hAccEffFD
 
 minimisation:
-    correlated: true #true --> fully correlated, false --> fully uncorrelated        
+    correlated: true #true --> fully correlated, false --> fully uncorrelated (if false, RawYieldSmearing automatically enabled)
```

Comment:
Thanks for spotting it! This is indeed a bug: the smearing and the correlation treatment should be independent